src/formio/components/radio.ts

```diff
@@ -20,23 +25,23 @@ interface BaseRadioSchema {
  * @group Form.io components
  * @category Base types
  */
-type RadioManualValuesSchema = Omit<RadioInputSchema, 'hideLabel' | 'disabled'> &
+type RadioManualValuesSchema = Omit<RadioInputSchema<ManualValues>, 'hideLabel' | 'disabled'> &
   BaseRadioSchema & {
-    openForms: OFExtensions<TranslatableKeys>['openForms'] & ManualValues;
     values: Option[];
   };
 
 /**
  * @group Form.io components
  * @category Base types
  */
-type RadioVariableValuesSchema = Omit<RadioInputSchema, 'hideLabel' | 'disabled'> &
-  BaseRadioSchema & {
-    openForms: OFExtensions<TranslatableKeys>['openForms'] & VariableValues;
-  };
+type RadioVariableValuesSchema = Omit<RadioInputSchema<VariableValues>, 'hideLabel' | 'disabled'> &
+  BaseRadioSchema;
 
 /**
  * @group Form.io components
  * @category Concrete types
  */
-export type RadioComponentSchema = RadioManualValuesSchema | RadioVariableValuesSchema;
+export type RadioComponentSchema = Require<
+  RadioManualValuesSchema | RadioVariableValuesSchema,
+  'openForms'
+>;
```

Comment:
Does the `Require` here mean that the `openForms` key is required for `RadioComponentSchema`?

Reply:
indeed! I found it a bit weird that there's no built-in utility type for it (`Required` makes everything required). Without this, the formio builder doesn't build properly, and for select, selectboxes and radio the `dataSrc` *must* be specified.